### ISSUE TITLE
Error: plugin failed to load

### ISSUE THREAD

**Issue by aurdszekely** [markdown-confluence/markdown-confluence#642]
Dear all,

 Recently, I’ve encountered an issue with the addon not loading in Obsidian. While I can’t pinpoint exactly when this problem started, it appears to be related to a newer version of Obsidian.

Upon inspecting the developer tools, I noticed the following error message:

`app.js:1 Plugin failure: confluence-integration TypeError: W_e.isDesktopCapturerEnabled is not a function
    at eval (plugin:confluence-integration:44:184470)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:184861)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:190951)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:190998)
    at eval (plugin:confluence-integration:6:265)
    at anonymous (plugin:confluence-integration:1718:226)
    at e.<anonymous> (app://obsidian.md/app.js:1:2243790)
    at app://obsidian.md/app.js:1:237056
    at Object.next (app://obsidian.md/app.js:1:237161)
    at a (app://obsidian.md/app.js:1:235879)
(anonymous) @ app.js:1`

I’ve attempted both a reinstall and a beta installation via BRAT (which seems to be the same version), but unfortunately, the issue persists. Notably, I don’t believe it’s related to Confluence, as the plugin remains disabled.

For reference, here are some details about my environment:

Operating System: Windows 11
Obsidian Version: 1.5.12
Confluence-Integration Plugin Version: 5.5.2

Any assistance or insights would be greatly appreciated.

**Comment by Danl2620** [comment]
I just installed the plugin and I'm seeing the same exact issue.

```
app.js:1 Plugin failure: confluence-integration TypeError: W_e.isDesktopCapturerEnabled is not a function
    at eval (plugin:confluence-integration:44:184470)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:184861)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:190951)
    at eval (plugin:confluence-integration:6:265)
    at eval (plugin:confluence-integration:44:190998)
    at eval (plugin:confluence-integration:6:265)
    at anonymous (plugin:confluence-integration:1718:226)
    at e.<anonymous> (app.js:1:2243790)
```

My show debug info:

```
SYSTEM INFO:
	Obsidian version: v1.5.12
	Installer version: v1.5.8
	Operating system: Windows 10 Enterprise 10.0.19045
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Obsidianite v2.0.2
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 8
	Plugins enabled: 7
		1: Templater v1.18.1
		2: Calendar v1.5.10
		3: Kanban v1.5.3
		4: Dataview v0.5.64
		5: Outliner v4.8.0
		6: Obsidian Git v2.22.0
		7: Version History Diff v2.1.9

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by StefanD986** [comment]
Probably a duplicate of https://github.com/markdown-confluence/markdown-confluence/issues/627
There is a workaround in the comments

**Comment by aurdszekely** [comment]
Hope this project is not dead, as the plugin is really useful. I implemented the workaround to delete the mentioned function call directly in the main.js file of the plugin. Its working again...

As this issue is already addressed with #627 , this one can be closed

Thanks for your help!
